### DISCUSSION TITLE
Date/time not displaying after upgrade to Ubuntu 24.04.1 LTS

### DISCUSSION THREAD

**Discussion by apoco** [regolith-linux/regolith-desktop#1056] [General]
Hi there. I did an upgrade to Ubuntu Noble Numbat (24.04.1), and strangely the date/time bar is not showing up. I tried uninstalling and re-installing `i3xrocks-time` to no avail. I also tried running `regolith-control-center` and setting the date/time settings, but that still didn't get it to show up.

Here's a variety of config files in case it's helpful. My `/etc/i3xrocks.conf`:

```ini
# Global properties
separator=true
separator_block_width=15

[documentation]
full_text=Documentation
website=https://vivien.github.io/i3blocks
command=xdg-open "$website"
color=#f12711

[greetings]
color=#f5af19
command=echo "Hello, $USER!"
interval=once

[time]
command=date '+%Y-%m-%d %H:%M:%S'
interval=1
```

`/etc/i3status.conf`:

```ini
general {
        colors = true
        interval = 5
}

order += "ipv6"
order += "wireless _first_"
order += "ethernet _first_"
order += "battery all"
order += "disk /"
order += "load"
order += "memory"
order += "tztime local"

wireless _first_ {
        format_up = "W: (%quality at %essid) %ip"
        format_down = "W: down"
}

ethernet _first_ {
        format_up = "E: %ip (%speed)"
        format_down = "E: down"
}

battery all {
        format = "%status %percentage %remaining"
}

disk "/" {
        format = "%avail"
}

load {
        format = "%1min"
}

memory {
        format = "%used | %available"
        threshold_degraded = "1G"
        format_degraded = "MEMORY < %available"
}

tztime local {
        format = "%Y-%m-%d %H:%M:%S"
}
```

What could be the cause? Any idea how I can debug what's going on?

**Comment by apoco** [comment]
By the way, adding `"order += "time"` did cause the time to show up; perhaps something changed with `i3status` where `tztime` doesn't work anymore?

**Comment by ethanfel** [comment]
thanks for the info, I just upgraded from to 22.04 to 24.04 and time is gone too. I don't even have a  /etc/i3status.conf , for me It was that most i3xrocks module were removed instead of upgraded after the upgrade

**Comment by spuro** [comment]
I would assume something to do with this:
https://discourse.ubuntu.com/t/ubuntu-24-04-lts-noble-numbat-release-notes/39890#tzdata-package-split

I never used tztime pre 24, was it it's own command before? Only tzselect appears in the terminal in my fresh 24 install.

**Comment by Torchtopher** [comment]
Has anyone found a fix for this? Time is also gone for me after upgrading to 24.04. (also does not seem specific to regolith, i am using stock ubuntu)
